src/App.js

```diff
@@ -14,11 +14,51 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    good: 'Jam',
+  }
+
+  render() {
+    const { good } = this.state;
+
+    return (
+      <div className="App">
+        <h1>{`Selected good: ${good} is selected`}</h1>
+
+        <button
+          className={good === 'Nothing is selected' ? 'selectedButton' : ''}
+          type="button"
+          onClick={() => this.setState({ good: 'Nothing is selected' })}
+        >
+          clear
+        </button>
+
+        <br />
+        <br />
+
+        <ul>
+          {goodsFromServer.map(item => (
+            <li className={good.includes(item) ? 'yellow' : ''}>
```

Comment:
you can create some variable (`isIncluded` for example) inside callback in `map` method before you return `<li>`, and use this variable in all places where you need it. with this approach, you can use the method includes only once